molecule/default/molecule.yml

```diff
@@ -23,6 +23,8 @@ platforms:
       - 0.0.0.0:8080:8080/TCP
 provisioner:
   name: ansible
+  env:
+    ANSIBLE_VERBOSITY: 3
```

Comment:
@RanabirChakraborty I think this needs to be removed, no ?

Reply:
fixed it now!